### PR TITLE
Sam Peripherals

### PR DESCRIPTION
**Pull request by smasher816**

### CODE REVIEW

**Comment by smasher816** [comment]
Rebased and typo fixed

File: Scan/Devices/STLcd/lcd_scan.c [hunk 1]
```diff
@@ -498,6 +567,7 @@ void LCD_layerStackExact_capability( TriggerMacro *trigger, uint8_t state, uint8
 
 	// Number data for LCD
 	const uint8_t numbers[10][128] = {
+#if 0 //REMOVE ME
```
Comment (haata) [inline Scan/Devices/STLcd/lcd_scan.c]: Needs removing?

File: Scan/Devices/STLcd/lcd_scan.c [hunk 2]
```diff
@@ -508,10 +578,12 @@ void LCD_layerStackExact_capability( TriggerMacro *trigger, uint8_t state, uint8
 		{ STLcdNumber7_define },
 		{ STLcdNumber8_define },
 		{ STLcdNumber9_define },
+#endif
 	};
 
 	// Color data for numbers
 	const uint16_t colors[10][3] = {
+#if 0 //REMOVE ME
```
Comment (haata) [inline Scan/Devices/STLcd/lcd_scan.c]: Needs removing?

**Comment by smasher816** [comment]
Should hopefully be ready to merge now

**Pull request merged**